README.md

```diff
@@ -358,6 +369,65 @@ final class MyReactor: Reactor {
 }
 ```
 
+### Pulse
+
+`Pulse` has diff only when mutated
+To explain in code, the results are as follows.
+```swift
+var messagePulse: Pulse<String?> = Pulse(wrappedValue: "Hello tokijh")
+
+let oldMessagePulse: Pulse<String?> = message
+message = "Hello tokijh"
+
+oldMessagePulse != messagePulse // true
+oldMessagePulse.value == messagePulse.value // true
+```
+
+Use when you want to receive an event only if the new value is assigned, even if it is the same value.
+like `alertMessage` (See follows or [PulseTests.swift](https://github.com/ReactorKit/ReactorKit/blob/master/Tests/ReactorKitTests/PulseTests.swift))
+```swift
+// Reactor
+private final class MyReactor: Reactor {
+  struct State {
+    @Pulse var alertMessage: String?
+  }
+
+  func mutate(action: Action) -> Observable<Mutation> {
+    switch action {
+    case let .alert(message):
+      return Observable.just(Mutation.setAlertMessage(message))
+    }
+  }
+
+  func reduce(state: State, mutation: Mutation) -> State {
+    var newState = state
+
+    switch mutation {
+    case let .setAlertMessage(alertMessage):
+      newState.alertMessage = alertMessage
+    }
+
+    return newState
+  }
+}
+
+// View
+reactor.state.pulse(\.$alertMessage)
```

Comment:
I think `reactor.state.pulse(\.$alertMessage)` is a typo of `reactor.pulse(\.$alertMessage)`
Is there any other way to use it?

Reply:
Oops! Thank you:)